### ISSUE TITLE
NoodleMagazine Resolver Issue Maybe It's Just Me Idk? Other Sites No Longer Working Like They Used To....

### ISSUE THREAD

**Issue by ohmichael** [dobbelina/repository.dobbelina#1180]
Title says the bug what I'm experiencing - the video list shows the videos but when you go to play the video it says error in the upper corner then closes out after a second. But I know it's a resolver/playback problem on noodlemagazine and some other sites other people haven't talked about yet messing up are messing up like either the list won't show videos or playback problems I went through all of yours other day and half of the sites worked half didn't - seemed like the main popular ones weren't working - porntrex was the only popular one that worked for me that's considered popular - the less popular ones worked but didn't pull up anything I cared for to see. Popular ones like uflash and upornia are fine now but sites like spankbang get cloudflare errors for the captcha it says from the reading of the tiny box in the corner - I thoroughly tested your addon as I love it and that's what I came up with on my end. I'd go through and fix on your time of course the sites that are no longer working that's what I'd do. I currently don't have a list of the non working sites but I know noodlemagazine has a playback issue and spankbang has a cloudflare problem I should of written everything down other day but forgot ;/ Also thank you for fixing the popular site pornhoarder - its a blessing to me you fixed it.

**Comment by EvelK1203** [comment]
I've been having troubles getting any playback with that site.

https://paste.kodi.tv/gowimolezi

**Comment by dobbelina** [comment]
I get errors to on NoodleMagazine + thumbnails don't load, sometimes only one or two.

**Comment by ErosVece** [comment]
Noodlemagazine video playback is fixed: https://github.com/dobbelina/repository.dobbelina/commit/78db58a5c5e5b200b6dc27c8684e2c61ca797a41

I've seen the missing thumbnails before, I tried to fix it then, and I tried to look at it just now.
And I have currently no idea why it isn't working.
When I turn on Fiddler Everywhere, to see the requests made by Kodi and all, the images load fine.
So for now, install Fiddler and you have images 🤣
